src/components/TableData.js

```diff
@@ -0,0 +1,13 @@
+import React, { useState } from "react";
+
+import TableRow from "./TableRow";
+function TableData(props) {
+  return (
+    <tbody>
+      {props.results.map((data, index) => {
+        return <TableRow key={index} data={data} />;
+      })}
```

Comment:
Since you have a new `TableRow` component, I think there is no point to have `TableData` component. I would move this code to `SearchResults` so the props are not passed through too many levels.